docs/pages/admin-guides/access-controls/access-lists.mdx

```diff
@@ -5,7 +5,7 @@ layout: tocless-doc
 ---
 
 Access Lists allow Teleport users to be granted long term access to resources
-managed within Teleport. With Access Lists, administrators and access list
+managed within Teleport. With Access Lists, administrators and Access List
```

Comment:
Do we have anything in place to prevent this kind of mistakes from being introduced in the future? AFAIK Vale is an optional linter at the moment.

Reply:
The linter is optional, though I'm planning to look into making it mandatory this quarter.